packages/ListBoxWithTags/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@paprika/list-box-with-tags",
+  "version": "0.0.1",
+  "description": "ListBoxWithTags behave as a ListBox with the difference that the selected options stacked on pills next to each other",
```

Comment:
Since there is no `Tag` component, maybe this should be `ListBoxWithPills`?
Maybe they dont know about the `ListBox`, so we shouldn't reference it in the description?

```
The ListBoxWithTags component allows users to select one or more options from a list contained within the component, showing a pill next to each.
```

As discussed, maybe we don't even need a separate component... it's still just a `ListBox`

Reply:
Remove ListBowWithTags as a package and put it inside of ListBox, the dev experience has increased I think.